grakn-test/src/test/java/ai/grakn/test/engine/tasks/manager/StandaloneTaskManagerTest.java

```diff
@@ -142,6 +145,9 @@ public void testStopSingle() {
 
     @Test
     public void consecutiveStopStart() {
+        // Disable excessive logs for this test
+        ((Logger) LoggerFactory.getLogger(StandaloneTaskManager.class)).setLevel(Level.WARN);
```

Comment:
If we are going to keep doing this programmatically, can we add a handy static method so that we don't need to keep remembering the correct syntax?